src/display/process/allProcs.go

```diff
@@ -283,11 +296,52 @@ func AllProcVisuals(dataChannel chan []*proc.Process,
 						runAllProc = true
 						killSelected = false
 						updateProcs()
+						sendSignal = false
+						updateUI()
+					case "j", "<Down>":
+						signals.Table.ScrollDown()
+						ui.Render(signals)
+					case "k", "<Up>":
+						signals.Table.ScrollUp()
+						ui.Render(signals)
+					case "0", "1", "2", "3", "4", "5", "6", "7", "8", "9":
+						/*
+						* The signal selector can be navigated by entering the number beside the
+						* desired signal. Double digit numbers are handled by checking the previous
+						* key and, if it is among 1,2 and 3, navigate to the corresponding double
+						* digit number (as there are currently 31 supported signals).
+						* For example, pressing 25 would first navigate to signal 2, then to signal 25
+						 */
+						scrollIdx, _ := strconv.Atoi(e.ID)
+						if _, checkPrev := map[string]bool{"1": true, "2": true, "3": true}[previousKey]; checkPrev {
+							prevIdx, _ := strconv.Atoi(previousKey)
```

Comment:
(nitpick) since it's just comparing with 3 numbers, won't a simple if-statement be better here?

Converting it to an int before the comparison will also be (negligibly) faster.

```suggestion
						prevIdx, _ := strconv.Atoi(previousKey)
						if prevIdx == 1 || prevIdx == 2 || prevIdx == 3 {
							
```

IMO it also looks cleaner.

Reply:
Yeah it does. This was a messy solution to my attempt to do this like I would in Python :smile:. One thing to consider, though, is that we may add more signals in the future. `htop`, for example, has 62 signals (I think?). Not sure if this approach would look as clean for 3 more cases (or the current approach, for that matter)

Reply:
> `htop`, for example, has 62 signals

Woah. TIL. I never realized the signal window was scrollable xD. Looks like it has 64 signals.

You're right, this won't look very good when there's 6 of them.